elements/lisk-p2p/src/p2p.ts

```diff
@@ -178,6 +180,7 @@ export class P2P extends EventEmitter {
 	private readonly _handleInboundSocketError: (error: Error) => void;
 	private readonly _peerHandshakeCheck: P2PCheckPeerCompatibility;
 
+	// tslint:disable-next-line: cyclomatic-complexity
```

Comment:
I would avoid disabling the cyclomatic-complexitiy. This should be our boundary to find a simpler way or breaking the function into pieces.

Reply:
I agree however my additions to this file in this PR are minimal, and cyclomatic complexity only popped up after merging feature branch. I would like to suggest we tackle this in a separate PR.

Reply:
Agree. Please, open an issue/task to not to forget and link it to this comment.